From a4491ac9103c6e02631fce120594e5797130271d Mon Sep 17 00:00:00 2001
From: Wolf <wolf@wolfsden.cz>
Date: Wed, 20 Jul 2022 02:00:53 +0200
Subject: [PATCH 3/4] Make generate_bash_completion compatible with busybox

Under busybox, expr does not accept -- (since it is not defined to take
any options, I would say it makes sense), so remove it from the script
generating bash completion.
---
 scripts/generate_bash_completion.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/generate_bash_completion.sh b/scripts/generate_bash_completion.sh
index 778810570c..46a3965f85 100755
--- a/scripts/generate_bash_completion.sh
+++ b/scripts/generate_bash_completion.sh
@@ -34,7 +34,7 @@ die() {
 }
 
 get_optarg() {
-  expr -- "${1}" : "[^=]*=\\(.*\\)"
+  expr "${1}" : "[^=]*=\\(.*\\)"
 }
 
 append=
-- 
2.36.2

